src/LoginComponent.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class LoginComponent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: ''
+    };
+  }
+
+  handleChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    this.loginUser();
+  }
+
```

Comment:
The loginUser function directly interacts with the axios library, which makes the component tightly coupled with the axios library. Consider abstracting the API calls into a separate API layer for better modularity and maintainability.